config/jobs-chromeos.yaml

```diff
@@ -679,6 +695,87 @@ jobs:
         - collabora-next:for-kernelci
     kcidb_test_suite: kselftest.device_error_logs
 
+  ltp-containers-cros-kernel:
+    <<: *ltp-cros-kernel-job
+    params:
+      <<: *ltp-cros-kernel-params
+      tst_cmdfiles: "containers"
+
+  ltp-controllers-cros-kernel:
+    <<: *ltp-cros-kernel-job
+    params:
+      <<: *ltp-cros-kernel-params
+      tst_cmdfiles: "controllers"
+
+  ltp-crypto-cros-kernel:
+    <<: *ltp-cros-kernel-job
+    params:
+      <<: *ltp-cros-kernel-params
+      tst_cmdfiles: "crypto"
+    rules:
+      fragments:
+        - 'crypto'
```

Comment:
This will overwrite the rules entirely, should likely be the following instead:

```yaml
    rules:
      fragments:
        - 'crypto'
        - '!kselftest'
      tree:
        - chromiumos
```

Or even better:

```yaml
    rules:
      <<: *ltp-cros-kernel-rules
      fragments:
        - 'crypto'
        - '!kselftest'
```